### ISSUE TITLE
Codes API

### ISSUE THREAD

**Issue by hparcells** [reverse-squared/card-craze#14]
# Codes API
> Possibly going to be renamed.

## Overview
There will be a table stored in RethinkDB called codes, which will contain all the codes. Codes are only redeemable with people that have accounts. Codes stored in the database will have a similar format to:

```
{
  code: "10characterCode",
  redeemed: false,
  redeemedBy: "email",
  type: "admin"
},
{
  code: "10characterCode",
  redeemed: true,
  redeemedBy: "email"
  type: "beta"
}
```

## To Do List
- [ ] Function to add codes to database.
- [ ] #12 Code Redeemer.
- [ ] Add codes by using commands from the Discord Bot

**Comment by imdaveead** [comment]
i think codes should not be a easy, guessable string. we could use uuids for generating codes, or a variation of that.

**Comment by hparcells** [comment]
Yeah, I was thinking it could be a 10 character alphanumeric code like `HP846rZTSH`.
